concepts/privacy.mdx

```diff
@@ -1,21 +1,25 @@
-Privacy in blockchain transactions is crucial, and mev-commit directly addresses this. It protects bidders' strategic interests by keeping bid and commitment details confidential. This approach is vital in preventing potential manipulations in the auction process. mev-commit provides end-to-end privacy for all bids and commitments in the sense that until commitments are opened after the corresponding L1 block have been confirmed, the contents of the commitment and the corresponding bid are only visible to the bidder who made the bid and the provider who made the commitment. All other bidders and providers and external parties learn nothing except that this provider has made a commitment. Below we expand on the privacy notions achieved for the bidders and the providers.
+If bids and transaction bundles are transmitted in clear text over the peer-to-peer (P2P) network, it could allow competing bidders and providers to observe and adjust their strategies based on the visible bids. This visibility potentially harms the expected utility of bidders, as their competitors could outbid them by adjusting their bids accordingly. Moreover, the premature revelation of a provider’s commitment to a bid could lead to information leakage about the total value of the block being constructed. This not only affects the privacy of the providers but also skews the competitive landscape, enabling providers to underbid based on the leaked value information.
+To counteract these privacy issues mev-commit provides end-to-end privacy for all bids and commitments in the sense that until commitments are opened after the corresponding L1 block have been confirmed, the contents of the commitment and the corresponding bid are only visible to the bidder who made the bid and the provider who made the commitment. All other bidders and providers and external parties learn nothing except that this provider has made a commitment. Moreover, it is crucial that commitments to bids are binding to the specific bid and that it hides its contents until the opening of the commitment. 
```

Comment:
"the corresponding L1 block have been confirmed" -> has been confirmed

Reply:
"and that it hides its contents until the opening of the commitment." This seems redundant since hiding is already discussed before.